sdk/identity/azcertificatemanager/README.md

```diff
@@ -0,0 +1,69 @@
+# Azure Template Package client library for Go
```

Comment:
You'll need to update the content of this file.

Reply:
@priyankatiwaridixit : Can you advise which document to refer to address this? Normal syntax concerns are resolved, but this pertains definition of our product.

Reply:
Added relevant document for details for now. If needed will add specific commands at public preview.